specification/ai/Azure.AI.Unified/datasets/routes.tsp

```diff
@@ -0,0 +1,132 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../common/models.tsp";
+import "./models.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+using Azure.Core.Foundations;
+
+namespace Azure.AI.Unified;
+
+interface Datasets {
+  #suppress "@azure-tools/typespec-azure-core/operation-missing-api-version"
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("List datasets in a project.")
+  @route("/datasets")
+  @get
+  listDatasets(
+    @doc("Continuation token for pagination.")
+    @query("$skip")
+    $skip?: string,
+
+    @doc("View type for including/excluding (for example) archived entities.")
+    @query("listViewType")
+    listViewType?: ListViewType,
+  ): PagedDatasetContainer;
+
+  #suppress "@azure-tools/typespec-azure-core/operation-missing-api-version"
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Delete container.")
+  @route("/datasets/{name}")
+  @delete
+  delete(
+    @doc("Name of Azure Machine Learning workspace.")
+    @pattern("^[a-zA-Z0-9][a-zA-Z0-9_-]{2,32}$")
+    @path
+    workspaceName: string,
```

Comment:
Should this not be the AI Foundry project name? And how come we don't see this `workspaceName` on any other operation here?

Reply:
should be removed now